_features/lin_advance.md

```diff
@@ -19,11 +19,43 @@ Since the root cause is pressure, `LIN_ADVANCE` de-couples extrusion from the ot
 ## Advantages
 
 - Better dimensional precision due to reduced bleeding edges.
-- Higher printing speeds are possible without any loss of print quality.
+- Higher printing speeds are possible without any loss of print quality - as long as your extruder can handle the needed speed changes.
 - Visible and tangible print quality is increased even at lower printing speeds.
 - No need for high acceleration and jerk values to get sharp edges.
 
 <br>
+
+## Special notes for v1.5
+
+### Changelog
+
+- K is now a meaningful value with the unit [mm of filament compression needed per 1mm/s extrusion speed] or [mm/mm/s].
+- Load inside stepper ISR reduced as no calculations are needed there any more. Instead, the extruder runs at a fixed speed offset during pressure adjustment. Therefore this version runs faster.
+- LIN_ADVANCE now respects hardware limitations set in Configuration.h, namely extruder jerk. If the pressure corrections would need faster adjustments than allowed by extruder jerk limit, the acceleration for this print move is limited to a value which allows to use extruder jerk speed as the upper limit.
+- The pressure adjustment moves doesn't lead to a rattling extruder as it was in v1.0 as the extruder is now running at a smooth speed instead of jerking between multiples of extruder print speed.
```

Comment:
"The pressure adjustment moves **doesn't** lead to a rattling" --> "The pressure adjustment moves **don't** lead to a rattling"

"as it was in v1.0 as the extruder" --> "as it was in v1.0: the extruder"